src/modules/trading/components/trading--confirm/trading--confirm.styles.less

```diff
@@ -138,7 +138,7 @@
 
 .TradingConfirm__error_message_container {
   background: fade(@color-darker-teal, 90%);
-  border: 1px solid @color-bright-teal;
+  border: 1px solid @color-negative;
```

Comment:
rem?

Reply:
I don't see an alternative for 1px in the code.

Reply:
0.0625rem